frontend/src/views/shop/EditShop.vue

```diff
@@ -286,6 +291,33 @@
                 </button>
             </template>
         </Modal>
+
+        <!-- Plugin Connection Modal -->
+        <Modal :show="showPluginModal" close-x-mark @close="closePluginModal">
```

Comment:
I have in PR #504 make the modal template globale which makes sense. I think there is also a part that makes sense to edit and add

Reply:
Fixed in https://github.com/FriendsOfShopware/shopmon/pull/509/commits/4f527d2aa0192ef24d9d819f73c58fd4cb9d4425